plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/messages.properties

```diff
@@ -10,6 +10,11 @@ CREATE_APP_ENGINE_RUNTIME_WIZARD_DESCRIPTION=The App Engine Standard runtime req
 CREATE_APP_ENGINE_RUNTIME_WIZARD_TITLE=App Engine Standard Runtime
 SERVICE_CONFLICTS=Cannot add "{0}" as project "{1}" has App Engine Service ID: {2}
 OPEN_CLOUD_SDK_PREFERENCE_BUTTON=Open the Cloud SDK Location preference page when the wizard closes
+REVIEW_DATASTORE_INDEXES_UPDATE_MESSAGE=The local development server has updates for the \
```

Comment:
Per Google UI guidelines, "Avoid the subjunctive  'Do you want to save your changes?' is simpler to read than 'Would you like to save your changes?' The subjunctive rarely adds meaning, but is a symptom of elevated diction (using phrases, vocabulary, or syntax that is meant to impress as much as to communicate). Of course, even shorter and better than those examples is just, 'Save now?'"

So maybe "Review and merge the changes?"